packages/terra-table/docs/guides/SectionWithinSection.md

```diff
@@ -0,0 +1,144 @@
+# Terra Table - Sections with Subsections
+
+With the inclusion of sections in the table, there are two recommended patterns for created nested collapsible sections. First is collapsible sections and static subsections, and the second is static sections and collapsible subsections. The guidance from UX is to not have both sections and subsections to be collapsible, have one or the other be collapsible.
```

Comment:
> The guidance from UX is to not have both sections and subsections to be collapsible, have one or the other be collapsible.

The guidance from UX is to not have both collapsible sections and collapsible subsections in a single table.